runtime/lua/vim/lsp.lua

```diff
@@ -2048,7 +2048,7 @@ end
 ---@private
 ---@deprecated
 function lsp.get_active_clients(filter)
-  -- TODO: add vim.deprecate call after 0.10 is out for removal in 0.12
+  vim.deprecate('vim.lsp.get_active_cients', 'vim.lsp.get_clients', '0.12')
```

Comment:
This will be further disruptive for downstream users.

The `@deprecated` annotation should signal enough.

Reply:
but just use annotation not show `use vim.lsp.get_clients` instead of .